decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/presentation/external/ExtPacsController.java

```diff
@@ -46,8 +46,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(ExtPacsController.class);
 
     /**
-     * Validates an incomming pacs message before sending it to the internal
-     * financial institution system
+     * Validates a pacs message before sending it to the local financial institution system

```

Comment:
what does "**local** financial institution" refer to? Might be a bit ambiguous.

Reply:
It refers to whatever system will handle an incoming business message after being validated by the external component. Is there a better name for it, "message handling system"?

Reply:
I think "message handling system" describes it better, go with that instead 👍